src/app/molecules/message/Message.scss

```diff
@@ -168,7 +173,11 @@
   & > .text > a {
     white-space: initial !important;
   }
-  
+
+  & p:not(:last-child) {
+    margin-bottom: var(--sp-normal);
+  }
+
```

Comment:
Can we replace \n with <br /> instead of applying client specific css for newlines?

Reply:
this is to space `<p>` tags further apart to make them appear as actual paragraphs (as shown in the screenshots of #193). this does not relate to `\n` or `<br>` afaik.